### ISSUE TITLE
Creating 2.4.0 before 3.0.0 that has breaking changes

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2677]
How we handled this last time:

- disabling autoupdate,
- adding popup informing users about whet they need to do to continue using Quiet.

From the issue that we used last time (https://github.com/TryQuiet/quiet/issues/2039):

"When Quiet 2.x is released, we should do a final release of Quiet 1.x that has no changes, except that it shows a message reminding users that this release is not updated, that they should update Quiet by reinstalling from tryquiet.org, and that when they update their messages and community will not be carried over.

We can use the design for our "Software update" message, with the following text:

> Update available
> 
> Quiet’s next release makes joining communities faster and more reliable by letting people join when the owner is offline! 🎉 However, these changes are not backwards compatible, so you must re-install Quiet from tryquiet.org and re-create or re-join your community. 😥 This version of Quiet will no longer receive any updates or security fixes, so please re-install soon. We apologize for the inconvenience.
> 
> [Install Quiet 2.x]
> [Later]

This is not a normal autoupdate message. This message should appear on startup, forever.

The 1.x release should never update to 2.x (must use separate s3 buckets) and users should have to reinstall to get 2.x.

This release must happen after 2.x has been released."

**In our case all 1.x should be exchange for 2.x and 2.x for 3.x.** "Install Quiet 2.x" button directed users to the download section of the website so we should publish that only after our new prod version is published (contrary to what we talked about on daily!)

This is how it looked like:

![Screenshot 2024-12-19 at 15 58 57](https://github.com/user-attachments/assets/9279dbef-2215-4fed-b597-e3bec7792b6c)

**Comment by holmesworcester** [comment]
Duplicate of https://github.com/TryQuiet/quiet/issues/2672
